src/app/components/case/Documents/DocumentsTable/DocumentsTable.tsx

```diff
@@ -1,35 +1,34 @@
 import { useEffect, useMemo, useState } from "react"
 import { Table } from "@amsterdam/wonen-ui"
-import { useDocumentTypesByCase } from "app/state/rest"
 import getColumns from "./columns"
 
 type Props = {
   caseId: Components.Schemas.CaseEvent["id"]
-  data?: Components.Schemas.DocumentType[]
+  documents?: Components.Schemas.PaginatedDocumentTypeList["results"]
   loading: boolean
   getDocuments: () => Promise<unknown>
+  documentTypes?: Components.Schemas.DocumentType[]
 }
 
-const DocumentsTable: React.FC<Props> = ({ caseId, data, loading, getDocuments }) => {
+const DocumentsTable: React.FC<Props> = ({ caseId, documents, loading, getDocuments, documentTypes }) => {
   const [numLoadingRows, setNumLoadingRows] = useState(3)
-  const [documentTypes] = useDocumentTypesByCase(caseId)
 
   useEffect(() => {
-    if (data && data?.length > 0) {
+    if (documents && documents?.length > 0) {
```

Comment:
Dit is dubbel toch? 

Kan je hier niet schrijven: 
```suggestion
    if (documents?.length > 0) {
```
?

Reply:
Klopt! Die snap ik ook niet helemaal. Door het vraagteken doe je al een check op undefined. Helaas klaagt mijn IDE.